.travis/build-and-test-set-1.sh

```diff
@@ -9,9 +9,9 @@
   "project vector" test \
   "project vectortile" test \
   "project gdal" test \
-  "project geowave" compile test:compile \
+  # "project geowave" compile test:compile \
```

Comment:
Ouch in my experience comments like these were never interpreted properly by bash

Reply:
I know that it is still in progress, but I remembered about how the scala 2.12 / 2.11 dispatch happens now: https://github.com/locationtech/geotrellis/blob/master/.travis/build-and-test.sh 

`build-and-test-set-1` runs tests against Scala 2.11
We had to implement this "manual" scala versions dispatch due to geowave / geomesa 2.12 "support".

Do you think about moving GeoWave and GeoMesa packages out of this repo?

Reply:
Perhaps at least unlinking them from the multi-project build and having a direct dependency on a specific version of GT and its own test.

Reply:
My agenda is to get rid of these shell files as much as possible and do everything only with SBT commands. If we can pull that off that will reduce the number of places we introduce errors.

Reply:
this is 🔥